Google.Api.Gax.Grpc/Gcp/ChannelRef.cs

```diff
@@ -18,15 +19,29 @@ internal sealed class ChannelRef
     {
         private int _affinityCount;
         private int _activeStreamCount;
-        private int _id;
+        internal int _id;
```

Comment:
All fields should be private (other than constants).

If you need these to be accessed internally, they should be exposed as properties (and should probably be read only)